pkg/autoupdate/autoupdate.go

```diff
@@ -138,7 +140,7 @@ func createTUFRepoDirectory(localPath string, currentAssetPath string, assetDir
 			// files not yet yet there -- Generating an invalid state. Note:
 			// this does not check the validity of the files, they might be
 			// corrupt.
-			if _, err := os.Stat(fullAssetPath); !os.IsNotExist(err) {
```

Comment:
was this the bug that you mentioned related to tuf not replacing directories?

Reply:
Yes. It should have been `fullLocalPath` Might have only been broken if you were in the root of a launcher checkout. Not sure